README.md

```diff
@@ -87,10 +87,11 @@ Options:
                                   specified multiple times.
   --config FILE                   Read configuration from specified file.
                                   [default: tartufo.toml]
-  --local-config / --no-local-config BOOL
-                                  Enable or Disable processing of local config file
+  --target-config/--no-target-config BOOL
+                                  Enable or Disable processing of config file in the
+                                  repository being scanned
                                   i.e. config files like tartufo.toml or pyproject.toml
-                                  setup in local working directory
+                                  setup in the repository being scanned
```

Comment:
Does this flag apply to the `scan-folder` subcommand as well, or only `scan-local-repo` and `scan-remote-repo` ? What about `pre-commit` ? Depending on which it applies to, the wording could potentially be improved here.

Reply:
It should apply to both `scan-folder` and `pre-commit` along with `scan-local-repo` and `scan-remote-repo`

Reply:
Okay, I would argue that this should not say `the repository being scanned` then, since it applies to `scan-folder` too. Maybe just `the repository/folder` being scanned?

Reply:
```suggestion
                                  setup in the repository or folder being scanned
```